app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -278,8 +295,24 @@ class RoutePresenter(
         activity.startActivity(browserIntent, null)
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
     private fun openSetting(settingAction: RouteAction.SystemSetting) {
-        val settingIntent = Intent(settingAction.setting.intentAction)
-        activity.startActivity(settingIntent, null)
+        if (settingAction.setting == SettingIntent.Autofill) {
```

Comment:
Is there a better way of doing this than a) in the `openSetting()` method, or b) protecting the `openSetting` method with a Android O `RequiresApi` annotation?

Reply:
yeah, double-work from #417 for me.